app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,13 @@
+class StaticPagesController < ApplicationController
+  def home
+    # ...
+  end
```

Comment:
```suggestion
  def home; end
```
coding convention: ko có thì viết trên 1 line, đưa code comment vào làm j cu